cookbook/ssh/default.rb

```diff
@@ -0,0 +1,28 @@
+directory 'create .ssh directory' do
+  path "/home/rails/.ssh"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.